include/libsemigroups/sims.hpp

```diff
@@ -1973,9 +1969,19 @@ namespace libsemigroups {
     using SimsSettings<MinimalRepOrc>::stats;
 
     //! Default constructor
-    MinimalRepOrc() = default;
+    MinimalRepOrc() : _size() {
+      init();
+    }
 
-    // TODO(doc)
+    //! \brief Reinitialize an existing MinimalRepOrc object.
+    //!
+    //! This function puts a MinimalRepOrc object back into the same state as if
+    //! it had been newly default constructed.
+    //!
+    //! \returns A reference to \c this.
+    //!
+    //! \exception
```

Comment:
```suggestion
    //! \exceptions
```

See #628 for more info.